pkg/providers/docker/config/template.yaml

```diff
@@ -85,13 +85,42 @@ spec:
         certSANs:
         - localhost
         - 127.0.0.1
-{{- if .extraArgs }}
         extraArgs:
-{{ .extraArgs.ToYaml | indent 10 }}
+          anonymous-auth: "false"
+          audit-policy-file: /etc/kubernetes/audit-policy.yaml
+          audit-log-path: /var/log/audit.log
```

Comment:
Can we either rename this file or put it in a kubernetes-specific directory? If I'm looking in `/var/log` and see `audit.log` that could be confused with other things like the linux audit log.

Reply:
I can put it in `/etc/kubernetes`